blocklanguages.php

```diff
@@ -97,8 +57,8 @@ protected function _prepareHook($params)
 	*/
 	public function hookDisplayTop($params)
```

Comment:
Installer is not installing on Top.

https://github.com/PrestaShop/blocklanguages/pull/6/files#diff-4f8b40db08a06b786f3f1292c70170fcR49

Best regards